com.sap.cloud.lm.sl.cf.process/src/test/java/com/sap/cloud/lm/sl/cf/process/jobs/CleanUpJobTest.java

```diff
@@ -0,0 +1,27 @@
+package com.sap.cloud.lm.sl.cf.process.jobs;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.junit.Test;
+
+public class CleanUpJobTest {
+    private CleanUpJob cleanUpJob = new CleanUpJob();
+    
+    @Test
+    public void testSplitAllFilesInChunks() {
+        
+        Map<String, List<String>> spaceToFileIds = new HashMap<>();
+        List<String> fileIds = new ArrayList<String>();
+        fileIds.add(null);
+        fileIds.add("9f87be64-6519-4576-b426-42548840f2ec");
+        fileIds.add("9f87be64-6519-4516-b426-42543845f2az,9f87ne64-6519-1234-b426-42548840f2gh,9f87be64-1239-4567-b426-34548840f2oq");
+        spaceToFileIds.put("space", fileIds);
+        Map<String, List<String>> splitAllFilesInChunks = cleanUpJob.splitAllFilesInChunks(spaceToFileIds);
+        assertEquals(5, splitAllFilesInChunks.get("space").size());
```

Comment:
Here maybe an assertion, whether the files are the one that should be, will be good?

Reply:
Done.